presto-memory/src/test/java/io/prestosql/plugin/memory/TestMemorySmoke.java

```diff
@@ -307,7 +363,7 @@ private Session withPartitionedJoin()
         QueryStats stats = runner.getCoordinator().getQueryManager().getFullQueryInfo(queryId).getQueryStats();
         return stats.getOperatorSummaries()
                 .stream()
-                .filter(summary -> summary.getOperatorType().equals("ScanFilterAndProjectOperator"))
+                .filter(summary -> summary.getOperatorType().contains("Scan"))
```

Comment:
Make `assertDynamicFiltering` accept `ImmutableList`

Reply:
Sounds good, will update the code,